disk/disk.go

```diff
@@ -154,6 +154,15 @@ func (d *Disk) Remove(name string, all bool) error {
 	return os.RemoveAll(name)
 }
 
+func (d *Disk) Sync(name string) error {
```

Comment:
Why do we need a extra sync call? Should we just allow a write to optionally force a sync?